wagtail/admin/tests/test_pages_views.py

```diff
@@ -3002,9 +3003,14 @@ def test_page_copy_post_invalid_slug(self):
         self.assertEqual(response.status_code, 200)
 
         # Check that a form error was raised
-        self.assertFormError(
-            response, 'form', 'new_slug', "Enter a valid 'slug' consisting of letters, numbers, underscores or hyphens."
-        )
+        if DJANGO_VERSION >= (3, 0):
+            self.assertFormError(
+                response, 'form', 'new_slug', "Enter a valid “slug” consisting of letters, numbers, underscores or hyphens."
```

Comment:
https://unicode-table.com/en/201C/ vs the standard https://unicode-table.com/en/0022/
similar question about the change here. I understand why you didn't use the double quote, but is there something that changed which caused the `'` to be an invalid test?

Reply:
@ntravis All of these validation messages are defined by Django, not Wagtail. I assume that the Django team has simply decided to change these messages to use curly quotes in 3.0 for aesthetic reasons, and that this is unrelated to the change to the `escape` function.